esp8266/Makefile

```diff
@@ -148,7 +148,7 @@ DRIVERS_SRC_C = $(addprefix drivers/,\
 SRC_S = \
 	gchelper.s \
 
-FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py')
+FROZEN_MPY_PY_FILES := $(shell find -L $(FROZEN_MPY_DIR) -type f -name '*.py') modules/upip.py modules/upip_utarfile.py
```

Comment:
> 2 is simpler. How about untaring upip in lib/upip/ and just symlinking from there to {esp8266,unix,etc}/modules/?

What about building on windows, where there commonly wouldn't be symlinks support?

@stinos : Any comments on that?

And lib/upip/ somehow doesn't feel right, lib/ so far is used mostly by C code. I'd leave it in tools/ then.